packages/client/src/utils/gmodel-util.ts

```diff
@@ -416,3 +463,29 @@ export function isNotDescendantOfAnyElement<T extends GModelElement>(elements: F
 export function removeDescendants<T extends GModelElement>(elements: FluentIterable<T>): FluentIterable<T> {
     return elements.filter(isNotDescendantOfAnyElement(elements));
 }
+
+/**
+ * Computes the bounds of the given edge, i.e., the bounding box of the edge's source, target, and routing points in absolute coordinates.
+ *
+ * @param edge The edge to compute the bounds for.
+ * @returns the bounds of the edge
+ */
+export function getAbsoluteEdgeBounds(edge: GEdge): Bounds {
+    if (!edge.source || !edge.target) {
+        return edge.bounds;
+    }
+    const source = edge.source;
+    const target = edge.target;
+    const edgePoints: Bounds[] = [];
+    edgePoints.push({ ...Dimension.ZERO, ...Bounds.center(source.bounds) });
```

Comment:
Am I misunderstanding something, or is this the hard coded assumption that source and target are always anchored in center? Which would not be correct.

Reply:
Very good point! I'm not sure we can provide a generic method without handing in at least the edge registry. Since this is currently not used, I may just skip it. Thanks!